Connected/Assets/Prefabs/Container.prefab

```diff
@@ -83,83 +84,97 @@ MeshRenderer:
   m_SortingLayer: 0
   m_SortingOrder: 0
   m_AdditionalVertexStreams: {fileID: 0}
---- !u!54 &8907785965736572327
+--- !u!54 &601227700645040077
 Rigidbody:
   m_ObjectHideFlags: 0
   m_CorrespondingSourceObject: {fileID: 0}
   m_PrefabInstance: {fileID: 0}
   m_PrefabAsset: {fileID: 0}
-  m_GameObject: {fileID: 3880800136033850485}
+  m_GameObject: {fileID: 601227700645040078}
   serializedVersion: 2
   m_Mass: 2000
   m_Drag: 0
   m_AngularDrag: 0.05
   m_UseGravity: 1
-  m_IsKinematic: 0
+  m_IsKinematic: 1
   m_Interpolate: 0
   m_Constraints: 0
   m_CollisionDetection: 0
---- !u!1 &3967983900875730857
+--- !u!64 &1409008898
+MeshCollider:
```

Comment:
I don't understand why this solved the issue, but as long as it did, we can live with the performance issue.

Reply:
And by the way, the performance will never be a problem, it's just me being extremely picky

Reply:
I think this was just something I tested and is really not necessary. The actual fix was using OnTriggerEnter.

Reply:
I’ll separate the mesh into 5 pieces and have a box collider for each. That will do.

Reply:
Nah this works, leave it as it is :)

Reply:
If you say so :)